src/types/internal.types.ts

```diff
@@ -191,3 +191,10 @@ export interface Signature {
   r: string
   s: string
 }
+
+export interface WalletAnalyticsRequest {
+  wallet_address: string
+  fromTimestamp: number
+  toTimestamp: number
+  integrator?: string
+}
```

Comment:
Why do we end up with a type here again? I remember we moved it to the types package.
 Also, why do we start using snake_case?

Reply:
This is just the request type for the endpoint, but just checked that all the request types as in /types package.

moving it there.